### PR TITLE
Add `path` option

### LINKED ISSUE #11: Add `path` option
Idea from https://github.com/sindresorhus/wait-for-localhost/issues/10.

### PR DESCRIPTION
**Pull request by brandon93s**
Closes #11

### CODE REVIEW

**Comment by brandon93s** [comment]
An identical PR, adding a `host` option, makes this super flexible.  😉

**Comment by sindresorhus** [comment]
> An identical PR, adding a host option, makes this super flexible. 😉

Alright. Sure. Since it's so simple to add.

File: index.d.ts [hunk 1]
```diff
@@ -5,6 +5,11 @@ declare namespace waitForLocalhost {
 		 */
 		port?: number;
 
+		/**
+		@default '/''
+		 */
```
Comment (sindresorhus) [inline index.d.ts]: Needs a doc description. Same with the readme.

Comment (sindresorhus) [inline index.d.ts]:
```suggestion
		*/
```

**Review by sindresorhus — CHANGES_REQUESTED** [review]

File: index.d.ts (hunk 1 above)
Comment (bndnsmth) [reply index.d.ts]: Space character is on the existing options, will remove from all. 🎯

Comment (bndnsmth) [reply index.d.ts]: ✔️

Comment (bndnsmth) [reply index.d.ts]: ✔️

**Pull request merged**